buzzmobile/constants.yaml

```diff
@@ -11,7 +11,7 @@ pixels_per_m: 80
 # Used for tentacle projection, travel distance is the step between each 
 # tentacle point, and num_points is the length of each tentacle, if using
 # liner, and not tentacle projection, ignore
-travel_distance: 0.3 
+travel_distance: 0.061
```

Comment:
This is a magic number, please explain how this number was chosen if it wasn't arbitrary.